### DISCUSSION TITLE
WLAN AP still active even if turned off

### DISCUSSION THREAD

**Discussion by FauthD** [emsesp/EMS-ESP32#1046] [General]
Hi Proddy, 
just found this in my router (Fritz!Box). The ems-esp is still in AP mode (at least it is listed, even after refreshing the list).

My first idea was that this is an artefact of rebooting the ems-esp32. But after refreshing the list it shows up again.
I did not expect that it is even possible to use the AP mode and work as a client at same time.
AP settings is "never"  and UI says "inactive".

Any ideas?

![2021-03-16 18 41 59 192 168 168 1 e6fa0925b362](https://user-images.githubusercontent.com/19874899/111356856-0eb02b00-8689-11eb-8588-918e14845295.png)

![2021-03-16 18 47 50 ems-esp 13e6aa709a3b](https://user-images.githubusercontent.com/19874899/111356317-89c51180-8688-11eb-8160-bc912c5d5c00.png)

![2021-03-16 18 50 06 ems-esp 9a15e7888b12](https://user-images.githubusercontent.com/19874899/111356354-934e7980-8688-11eb-9ffd-900e6a79c20a.png)

Dieter Fauth

**Comment by proddy** [comment]
oh, ok. Let me check here too. Thanks for reporting!

**Comment by proddy** [comment]
I can't reproduce. Could be that the router keeps hold of it. Can you join the AP? Or ping or telnet too 192.168.4.1?

**Comment by FauthD** [comment]
Hi,
Could well be a bug  in my frotzbox 7590, v7.25, also seen with 7.21.
With other FB and Android devices I cannot find the AP.
Strange is that the FB shows the MAC of the AP (client mac +1).

Perhaps it apears for a short time after reboot.
Then it should be disapear on reboot of the FB, but it sticks.

Dieter

**Comment by tp1de** [comment]
I can't confirm. I am using FB7590 v7.25 as well.
But keep in mind that the mesh network overview in the FB takes some hours to find inactive devices.

**Comment by proddy** [comment]
Ok, can I close this and re-open if it happens again?

**Comment by FauthD** [comment]
Yes, please close.
So far I could not find the undesired AP with other tools.
Most likely some glitch in the FB.
Dieter :-)
